commands/playground/youtubeQueue.js

```diff
@@ -0,0 +1,30 @@
+const ytdl = require("ytdl-core");
+const {youtubeSearch}  = require("./../music/youtubeSearch");
+
+const youtubeQueue = (queue, connection, voiceChannel) => {
+    
+    youtubeSearch(queue[0]).then((ytvid) => {
+        console.log(`queue in search ${queue}`)
+        
+        var dispatcher = connection.playStream(ytdl(ytvid, {filter:"audioonly"}), {seek: 0, volume: 1});
+        dispatcher.setVolumeLogarithmic(5 / 5);
```

Comment:
5/5 == 1?

Reply:
idk I got it following the video tutorial. I have since changed it from .setVolumeLogarithmic to .setVolume

Reply:
just noticed its logarithmic, still doesnt explain that but maybe that explains the range of 20-200